test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/TagHelpersWebSite.Home.Environment.html

```diff
@@ -0,0 +1,78 @@
+﻿
```

Comment:
leftover file; revert it when you squash your commits or before if you send out another iteration